src/libs/dutil/test/DUtilUnitTest/DUtilTests.cpp

```diff
@@ -21,7 +21,13 @@ namespace DutilTests
             Dutil_TraceSetLevel(REPORT_DEBUG, FALSE);
 
             Action^ action = gcnew Action(this, &DUtil::CallDutilTraceErrorSource);
-            Assert::Throws<Exception^>(action);
+            // The following line is ambiguous when used with the current version of Xunit.
+            // Xunit::Assert::Throws<Exception^>(action);
+            // It has therefore been, at least temporarily, replaced by the next line of code.
+            // See the comments in WixBuildTools.WixAssert for details.
+            WixAssert::Throws<Exception^>(action);
+
+            WixBuildTools::TestSupport::WixAssert::Throws<Exception^>(action);
```

Comment:
Please remove this line and change `Void` back to `void`. I can take this, but I fully expect to have to put it back to skipped.